blocks/cards/cards.css

```
@@ -1,123 +1,141 @@
-  align-items: center;
-  border: 1px solid var(--clr-blue-700);
-  border-radius: 24px;
-  box-sizing: border-box;
-  background-color: var(--clr-neutral-200);
-  color: var(--clr-blue-700);
-  display: flex;
-  justify-content: center;
-  margin-top: auto;
-  padding: 12px 24px;
-  text-decoration: none;
-  width: 100%;
-  margin-bottom: 0;
+.cards.cta .cards-card-body a.button {
+    align-items: center;
+    border: 1px solid var(--clr-blue-700);
+    border-radius: 24px;
+    box-sizing: border-box;
+    background-color: var(--clr-neutral-200);
+    color: var(--clr-blue-700);
+    display: flex;
+    justify-content: center;
+    margin-top: auto;
+    padding: 12px 24px;
+    text-decoration: none;
+    width: 100%;
+    margin-bottom: 0;
 }
 
 .cards .cards-card-image {
-  line-height: 0;
+    line-height: 0;
 }
 
 .cards > ul > li img {
-  width: 100%;
-  aspect-ratio: 6/5;
-  object-fit: cover;
-  border-radius: 6px;
+    width: 100%;
+    aspect-ratio: 6/5;
+    object-fit: cover;
+    border-radius: 6px;
 }
 
 .cards.cta > ul > li img {
-  aspect-ratio: 16/9;
-  border-bottom-left-radius: 0;
-  border-bottom-right-radius: 0;
+    aspect-ratio: 16/9;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
 }
 
 @media (width >= 600px) {
-  .cards > ul {
-    grid-template-columns: 1fr 1fr 1fr;
-    grid-gap: 20px;
-  }
-
-  .cards.two-columns > ul {
-    grid-template-columns: 1fr 1fr;
-  }
-
-  .cards > ul > li {
-    margin-bottom: 32px;
-  }
-
-  .cards .cards-card-body h2,
-  .cards .cards-card-body h3 {
-    line-height: 44px;
-  }
-
-  .cards .cards-card-body {
-    padding: 16px 16px 0;
-  }
-
-  .cards > ul > li img {
-    min-height: 182px;
-    min-width: 219px;
-    object-fit: cover;
-    aspect-ratio: 6/5;
-    border-bottom-left-radius: 6px;
-    border-bottom-right-radius: 6px;
-  }
+    .cards > ul {
+        justify-content: space-between;
+    }
+
+    .cards > ul > li {
+        width: calc(100% / 3 - 12px);
+        margin-bottom: 32px;
+    }
+
+    .cards.two-columns > ul > li,
+    .cards.cta > ul > li {
+        width: calc(100% / 2 - 12px);
+    }
+
+    .cards.cta > ul > li:nth-last-child(1):nth-child(odd) {
+        margin: 0 auto;
+    }
+
+    .cards.two-columns.cta > ul > li:nth-last-child(1):nth-child(odd) {
+        margin: 0 0 32px;
+    }
+
+    .cards .cards-card-body h2,
+    .cards .cards-card-body h3 {
+        line-height: 44px;
+    }
+
+    .cards .cards-card-body {
+        padding: 16px 16px 0;
+    }
+
+    .cards > ul > li img {
+        min-height: 182px;
+        min-width: 219px;
+        object-fit: cover;
+        aspect-ratio: 6/5;
+        border-bottom-left-radius: 6px;
+        border-bottom-right-radius: 6px;
+    }
 }
 
 @media (width >= 900px) {
-  .cards.cta {
-    margin: 0 100px;
-  }
-
-  .cards .cards-card-body {
-    padding-left: 32px;
-    padding-right: 32px;
-  }
+    .cards.cta > ul > li {
+        width: calc(100% / 3 - 12px);
```

```suggestion
        width: calc((100% / 3) - 12px);
```

I know order of operations, but just trying to clarify the intent for future readers of this code